app/models/podcast.rb

```diff
@@ -161,6 +161,18 @@ def categories=(cats)
     self[:categories] = sanitize_categories(cats, false).presence
   end
 
+  def locked=(val)
```

Comment:
Would maybe add a comment about why this works the way it does. Seems like it probably never gets used, but if it's going to exist, someone will come across it someday and probably wonder why we have a way to basically like podcasts permanently.